spec/controllers/editorial/news_controller_spec.rb

```diff
@@ -0,0 +1,80 @@
+require 'rails_helper'
+
+RSpec.describe Editorial::NewsController, type: :controller do
+
+  let!(:root_node) { Fabricate(:root_node) }
+  let!(:news) { Fabricate(:section, name: 'news') }
+  let!(:section_a) { Fabricate(:section) }
+  let!(:section_b) { Fabricate(:section) }
+  let!(:section_c) { Fabricate(:section) }
+  let!(:user) { Fabricate(:user, author_of: section_a, reviewer_of: section_b) }
+
+  describe 'GET #new' do
+    context 'when a user is authorised' do
+      before do
+        sign_in(user)
+        get :new
+      end
+
+      it { expect(assigns(:sections)).to eq([section_a, section_b]) }
+    end
+
+    context 'when a user is not authorised' do
+      before { get :new }
+
+      it { is_expected.to set_flash[:alert].to('You are not authorized to access this page.') }
+    end
+  end
+
+  describe 'POST #create' do
+    let(:post_action) {
+      post :create, node: {
+        name: 'Test',
+        section_id: section_a.id,
+        short_summary: 'foo',
+        summary: 'bar',
+        content_body: 'foobar',
+        release_date: Time.now.to_s,
+        section_ids: [section_b.id]
+      }
+    }
+
+    context 'when a user is authorised' do
+      before do
+        sign_in(user)
+        post_action
+      end
+
+      it 'should link to news_home_node as parent' do
```

Comment:
You could put the tests on the last node into its own describe block:

    describe 'the created node' do
      subject { Node.last }
      its(:parent) { is_expected.to eq(news.home_node) }
    end